fenix/app/src/main/java/org/mozilla/fenix/trackingprotection/ProtectionsStore.kt

```diff
@@ -109,6 +137,25 @@ data class ProtectionsState(
     }
 }
 
+/**
+ * COOKIE_BANNER_HANDLING_ENABLE - The site domain wasn't added to the list of exceptions.
+ * COOKIE_BANNER_HANDLING_DISABLE - The site domain was added to the exceptions list.
+ * The cookie banner reducer will not work.
+ * SITE_NOT_SUPPORTED -  The site domain isn't added to the exceptions list, but the cookie banner
+ * reducer didn't work.
+ * REQUEST_UNSUPPORTED_SITE_SUBMITTED - The user has submitted the domain of
+ * the site where the cookie banner reducer didn't work.
+ * COOKIE_BANNER_HANDLING_HIDE - All the cookie banner handling in the tracking panel is hidden.
+ */
+@Parcelize
```

Comment:
We should have a general documentation for the `enum` and over each constant add the documentation. For example https://github.com/mozilla-mobile/firefox-android/blob/a488c06ae9ff9d060517da08e2a2db94465dc871/android-components/components/browser/icons/src/main/java/mozilla/components/browser/icons/IconRequest.kt#L61-L80

Reply:
Done